stable/fairwinds-insights/values.yaml

```diff
@@ -365,6 +365,41 @@ postgresql:
       cpu: 75m
       memory: 256Mi
 
+timescale:
+
+  # -- Use the ephemeral Timescale chart by default
+  ephemeral: true
+  # -- SSL mode for connecting to the database
+  sslMode: require
+  # -- Secret name to use for Timescale Password
+  existingSecret: fwinsights-timescale
+    # -- Username to connect to Timescale with
+  postgresqlUsername: postgres
+  # -- Name of the Postgres Database
+  postgresqlDatabase: fairwinds_insights
+  service:
+    # -- Port of the Timescale Database
+    primary:
+      port: 5433
+  persistence:
+    # -- Create Persistent Volume with Timescale
+    enabled: true
+  replication:
+    # -- Replicate Timescale data
+    enabled: false
+  loadBalancer:
+    enabled: false
+    # -- Resources section for Timescale
+  resources:
+    limits:
+      cpu: 1
+      memory: 1Gi
+    requests:
+      cpu: 75m
+      memory: 256Mi
+  persistentVolumes:
+    data:
+      mountPath: "/var/lib/postgresql-timescale"
```

Comment:
do we need this part? I would expect this to be automatic

Reply:
Yeah looks like this is the default value, so we can remove

Reply:
I was wondering if this would conflict with the other postgres

Reply:
done